frontend/app/src/pages/defi/deposits/liquidity/index.vue

```diff
@@ -42,9 +42,7 @@ onMounted(() => {
       </v-btn-toggle>
     </div>
     <div>
-      <keep-alive>
-        <router-view />
-      </keep-alive>
+      <router-view />
```

Comment:
For me keep alive make it heavier, because it need to keep the state of opened page. 
The only downside is everytime I open the opened page, it will fetch some endpoint again, but I think it's okay because it calls with cache=true.